src/main/java/edu/kis/powp/jobs2d/command/JsonCommandImporter.java

```diff
@@ -0,0 +1,56 @@
+package edu.kis.powp.jobs2d.command;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class JsonCommandImporter implements CommandImporter {
+
+    private final static Logger logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
+
+    @Override
+    public DriverCommand importCommands(String filePath) {
+        try {
+            String content = new String(Files.readAllBytes(Paths.get(filePath)));
```

Comment:
unnecessary additional responsibility (SRP). The source of "content" aspect should be OCP.